API/ev3_timer.c

```diff
@@ -95,7 +95,7 @@ bool tmInitKernel()
     for (TimerInterval i = ti10ms; i < ti_count; i++)
     {
         // create a timer object
-        int tfd = timerfd_create(CLOCK_MONOTONIC, 0);
+        int tfd = timerfd_create(CLOCK_MONOTONIC, O_CLOEXEC);

```

Comment:
It is not guarranteed; however the older libc does not support `TFD_CLOEXEC` and on my local Ubuntu instalation `TFD_CLOEXEC` is defined as `O_CLOEXEC`. However I think that this could be avoided by a conditional preprocessor definition (so it is defined only when needed).

According to the manpage, this should work on 2.6.27+, which the EV3 kernel satisfies.

Reply:
Partially fixed in 78a52556814146f6b56efcae59593a8595df3a1f :+1: